frontend/src/main/components/Commons/ManageCows.js

```diff
@@ -1,15 +1,15 @@
-import React from "react";
-import { Card, Button, Row, Col } from "react-bootstrap";
-import cowHead from "./../../../assets/CowHead.png"; 
+import { Button, ButtonGroup, Card, Col, Row } from "react-bootstrap";
+import cowHead from "./../../../assets/CowHead.png";
 
-// add parameters 
-const ManageCows = ({userCommons,commons, onBuy, onSell}) =>  {
+
+// add parameters
+const ManageCows = ({userCommons, commons, onBuy, onSell}) =>  {
     // update cowPrice from fixture
+    // Stryker disable ArithmeticOperator : line 24: not very important and unsure of how to test this
```

Comment:
Can you explain more about what's going on here?   Stryker disable needs to be used very sparingly.